Tests/Packet++Test/main.cpp

```diff
@@ -41,48 +39,50 @@ int main(int argc, char* argv[])
 	bool memVerbose = false;
 	bool skipMemLeakCheck = false;
 
-	while((opt = getopt_long(argc, argv, "msvwht:x:", PacketTestOptions, &optionIndex)) != -1)
+	while ((opt = getopt_long(argc, argv, "msvwht:x:", PacketTestOptions, &optionIndex)) != -1)
 	{
 		switch (opt)
 		{
-			case 0:
-				break;
-			case 't':
-				userTagsInclude = optarg;
-				break;
-			case 'x':
-				userTagsExclude = optarg;
-				break;
-			case 's':
-				skipMemLeakCheck = true;
-				break;
-			case 'm':
-				memVerbose = true;
-				break;
-			case 'w':
-				PTF_SHOW_SKIPPED_TESTS(true);
-				break;
-			case 'v':
-				PTF_SET_VERBOSE_MODE(true);
-				break;
-			case 'h':
-				printUsage();
-				exit(0);
-			default:
-				printUsage();
-				exit(-1);
+		case 0:
+			break;
+		case 't':
+			userTagsInclude = optarg;
+			break;
+		case 'x':
+			userTagsExclude = optarg;
+			break;
+		case 's':
+			skipMemLeakCheck = true;
+			break;
+		case 'm':
+			memVerbose = true;
+			break;
+		case 'w':
+			PTF_SHOW_SKIPPED_TESTS(true);
+			break;
+		case 'v':
+			PTF_SET_VERBOSE_MODE(true);
+			break;
+		case 'h':
+			printUsage();
+			exit(0);
+		default:
+			printUsage();
+			exit(-1);
 		}
 	}
 
 	std::cout << "PcapPlusPlus version: " << pcpp::getPcapPlusPlusVersionFull() << std::endl
-	<< "Built: " << pcpp::getBuildDateTime() << std::endl
-	<< "Built from: " << pcpp::getGitInfo() << std::endl;
+	          << "Built: " << pcpp::getBuildDateTime() << std::endl
+	          << "Built from: " << pcpp::getGitInfo() << std::endl;
 
-	#ifdef NDEBUG
+#ifdef NDEBUG
 	skipMemLeakCheck = true;
-	std::cout << "Disabling memory leak check in MSVC Release builds due to caching logic in stream objects that looks like a memory leak:" << std::endl
-	<< "     https://github.com/cpputest/cpputest/issues/786#issuecomment-148921958" << std::endl;
-	#endif
+	std::cout << "Disabling memory leak check in MSVC Release builds due to caching logic in stream objects that looks "
+	             "like a memory leak:"
```

Comment:
it's still breaks the string for some reason 😕

Reply:
fixed.